setup.py

```diff
@@ -7,7 +8,8 @@
     use_setuptools()
     from setuptools import setup, find_packages, Command
 
-version = '0.2.0'
+module_file = open('ceph_installer/__init__.py').read()
+metadata = dict(re.findall("__([a-z]+)__\s*=\s*'([^']+)'", module_file))
```

Comment:
Whoops, this is better-implemented in #48 ... I'll drop this commit from my PR here.